compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -64,11 +64,24 @@ private[repl] class Rendering(parentClassLoader: Option[ClassLoader] = None) {
       myClassLoader
     }
 
+  /** Used to elide long output in replStringOf.
+   *
+   * TODO: Perhaps implement setting scala.repl.maxprintstring as in Scala 2, but
+   * then this bug will surface, so perhaps better not?
+   * https://github.com/scala/bug/issues/12337
+   */
+  private[repl] def truncate(str: String): String = {
+    val ncp = str.codePointCount(0, str.length) // to not cut inside code point
+    if ncp > MaxStringElements && ncp > 3 then
+      str.substring(0, str.offsetByCodePoints(0, MaxStringElements - 3)) + "..."
```

Comment:
I do not have a preference, but we could maybe use the character `"…"` instead of `"..."`. This is not a character that is often used in text and is clearly distinguishable from an output that actually ends with `...`.

Reply:
Good idea! I'll implement that in my next commit.

Reply:
I just tried the idea with special elipsis character instead of `"..."` but it is a bit less difficult to see esp. e.g. after special letters that connect, e.g. the Arabic letter beh, so I think it is a good idea to add `" truncated" as well.

Reply:
I've fixed it now. Ready to merge?